### ISSUE TITLE
`SlevomatCodingStandard.Classes.ClassMemberSpacing` false negative when multiple attributes are used

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#1737]
Given following file

```php
<?php 

class Test
{

    public function method1(): void
    {
    }
    #[FirstAttribute]
    public function method2(): void
    {
    }
    #[FirstAttribute]
    #[AnotherAttribute]
    public function method3(): void
    {
    }

}
```

I'm getting error only for `method1`, but not `method2`:

```
Line 6: Expected 1 blank line after method, found 0.
```

Removing one of the attributes fixes the issue.

**Comment by janedbal** [comment]
Looks like `SlevomatCodingStandard.Classes.MethodSpacing.IncorrectLinesCountBetweenMethods` has the same issue

**Comment by kukulich** [comment]
I'm not able to repreduce it: https://github.com/slevomat/coding-standard/commit/8bf0408a9cf30687d87957d364de9a3d5d00d948

Could you please provide a failing test?

**Comment by janedbal** [comment]
I think we can close this, [MethodSpacing is fixed](https://github.com/slevomat/coding-standard/commit/374ad13f74ac98df7f4c0781866bf4ac78ddeb01) and ClassMemberSpacing was not reproduced. If I ever encounter this problem again, I'll send failing test right away.
